models/articles.models.js

```diff
@@ -31,3 +31,34 @@ exports.selectArticles = () => {
     return result.rows;
   });
 };
+
+exports.checkArticleExists = (article_id) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+    .then((result) => {
+      if (result.rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          msg: "this article_id doesn't exist",
+        });
+      }
+
+      return result.rows;
+    });
+};
+
+exports.selectCommentByArticleId = (article_id) => {
+    return db
+      .query(
+        `SELECT * FROM comments WHERE article_id = $1
+        ORDER BY created_at DESC;`,
+        [article_id]
+      )
+
+      .then(( result ) => {
+        if(result.rows.length === 0) {
+          return Promise.reject({status: 200, msg: 'this article has no comments'})
```

Comment:
Here you can just send back the empty array for the 200 no comments. So you can remove the condition